src/binary.cpp

```diff
@@ -72,19 +74,21 @@ std::vector<unsigned char> DecodeBase64(const std::string &input) {
   unsigned char *out = &ret[0];
 
   unsigned value = 0;
-  for (std::size_t i = 0; i < input.size(); i++) {
+  for (std::size_t i = 0, cnt = 0; i < input.size(); i++) {
+    if (isspace(input[i])) continue; // skip newlines
```

Comment:
Add braces, even to this one-line "if" statement; and then put the "continue" on its own line